### ISSUE TITLE
Seems that task setup does not exist

### ISSUE THREAD

**Issue by theist** [capistrano-plugins/capistrano-safe-deploy-to#2]
this gem rely on setup task, but in current capistrano (3.2.1) that task is stripped causing deploy fail after the recommended setup

```
cap development deploy --trace
cap aborted!
Don't know how to build task 'setup'
/home/carlos/.rbenv/versions/2.0.0-p195/gemsets/uem-facultades/gems/rake-10.3.2/lib/rake/task_manager.rb:62:in `[]'
/home/carlos/.rbenv/versions/2.0.0-p195/gemsets/uem-facultades/gems/rake-10.3.2/lib/rake/task.rb:353:in `[]'
/home/carlos/.rbenv/versions/2.0.0-p195/gemsets/uem-facultades/gems/capistrano-3.2.1/lib/capistrano/dsl/task_enhancements.rb:5:in `before'
/home/carlos/.rbenv/versions/2.0.0-p195/gemsets/uem-facultades/gems/capistrano-safe-deploy-to-1.1.0/lib/capistrano/tasks/safe_deploy_to.rake:31:in `<top (required)>'
```

**Comment by bruno-** [comment]
Thanks for reporting the bug.
I just checked and it seems the `setup` task was actually never present in the capistrano 3.

I added the `setup` task hook so this plugin works better with [capistrano-postgres](https://github.com/bruno-/capistrano-postgresql) and [capistrano-unicorn-nginx](https://github.com/bruno-/capistrano-unicorn-nginx).
I didn't expect the "hook" will blow up for other users..

I'll fix soon.
